.travis.yml

```diff
@@ -1,8 +1,31 @@
-language: objective-c
-
 notifications:
   email:
     on_success: never
     on_failure: change
 
 script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'
+
+git:
+  depth: 10
```

Comment:
Oh certainly. What's with 10? Does 1 work well since we're not building on tags?